<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PalkkaApuri</class>
 <widget class="QWidget" name="PalkkaApuri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Palkat ja korvaukset</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0">
        <item row="6" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Päivärahat</string>
          </property>
          <property name="buddy">
           <cstring>paivarahaEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="KpEuroEdit" name="kmEdit">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Palkkiot</string>
          </property>
          <property name="buddy">
           <cstring>palkkioEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Luontoisedut</string>
          </property>
          <property name="buddy">
           <cstring>luontoisEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Kilometrikorvaukset</string>
          </property>
          <property name="buddy">
           <cstring>kmEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Lisät ja korvaukset</string>
          </property>
          <property name="buddy">
           <cstring>lisaEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="KpEuroEdit" name="luontoisEdit"/>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Ateriakorvaukset</string>
          </property>
          <property name="buddy">
           <cstring>ateriaEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KpEuroEdit" name="palkkaEdit"/>
        </item>
        <item row="7" column="1">
         <widget class="KpEuroEdit" name="ateriaEdit"/>
        </item>
        <item row="6" column="1">
         <widget class="KpEuroEdit" name="paivarahaEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Palkat</string>
          </property>
          <property name="buddy">
           <cstring>palkkaEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Loma-ajan ja 
sosiaalipalkat</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>lomaEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KpEuroEdit" name="palkkioEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="KpEuroEdit" name="lisaEdit"/>
        </item>
        <item row="3" column="1">
         <widget class="KpEuroEdit" name="lomaEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Työntekijöiden maksut</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Ennakonpidätykset</string>
          </property>
          <property name="buddy">
           <cstring>pidatysEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KpEuroEdit" name="pidatysEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Eläkevakuutusmaksut</string>
          </property>
          <property name="buddy">
           <cstring>elakeEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KpEuroEdit" name="elakeEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Työttömyysvakuutusmaksut</string>
          </property>
          <property name="buddy">
           <cstring>tvredit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KpEuroEdit" name="tvredit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Ay-jäsenmaksut</string>
          </property>
          <property name="buddy">
           <cstring>ayedit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KpEuroEdit" name="ayedit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Työnantajan maksut</string>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Sairausvakuutusmaksu</string>
          </property>
          <property name="buddy">
           <cstring>svedit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KpEuroEdit" name="svedit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Eläkevakuutusmaksut</string>
          </property>
          <property name="buddy">
           <cstring>tyeledit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KpEuroEdit" name="tyeledit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Pankkitili</string>
         </property>
         <property name="buddy">
          <cstring>tiliCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="TiliCombo" name="tiliCombo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Tililtä maksetaan</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="nettoLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KpEuroEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpeuroedit.h</header>
  </customwidget>
  <customwidget>
   <class>TiliCombo</class>
   <extends>QComboBox</extends>
   <header location="global">tools/tilicombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>palkkaEdit</tabstop>
  <tabstop>palkkioEdit</tabstop>
  <tabstop>lisaEdit</tabstop>
  <tabstop>lomaEdit</tabstop>
  <tabstop>luontoisEdit</tabstop>
  <tabstop>kmEdit</tabstop>
  <tabstop>paivarahaEdit</tabstop>
  <tabstop>ateriaEdit</tabstop>
  <tabstop>pidatysEdit</tabstop>
  <tabstop>elakeEdit</tabstop>
  <tabstop>tvredit</tabstop>
  <tabstop>ayedit</tabstop>
  <tabstop>svedit</tabstop>
  <tabstop>tyeledit</tabstop>
  <tabstop>tiliCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
